src/main/java/com/greenfox/tribes1/Resources/Gold.java

```diff
@@ -0,0 +1,8 @@
+package com.greenfox.tribes1.Resources;
+
+import lombok.Data;
+
+@Data
```

Comment:
Try not to use @Data, I know it contains everything what we need, but it's not to stable in some cases